### PR TITLE
Update non-python deps

### PR DESCRIPTION
**Pull request by EmilioPeJu**
- Update busybox and use ntpd in it instead of a separate package
- Update dropbear
- Update ethtool
- Update readline
- Update ncurses
- Update script to fetch dependencies in CI
- Disable kernel options related to DRM framework (no plans to play doom in panda yet)
- Remove some unuses packages

Fixes issue #30 
Requires https://github.com/dls-controls/rootfs/pull/16

### CODE REVIEW

**Comment by EmilioPeJu** [comment]
This change depends on the following change in rootfs: https://github.com/dls-controls/rootfs/pull/16

**Comment by EmilioPeJu** [comment]
This was successfully tested in panda `ts-mo-panda-01`

File: rootfs/network/parse-config [hunk 1]
```diff
@@ -141,12 +141,6 @@ server 2.pool.ntp.org
 server 3.pool.ntp.org
 EOF
     fi
-
-    cat <<EOF
-driftfile /tmp/ntp.drift
-pidfile /var/run/ntpd.pid
-logconfig -syncevents -sysevents
```
Comment (coretl) [inline rootfs/network/parse-config:148]: What does deleting these options mean?

File: rootfs/CONFIG [hunk 2]
```diff
@@ -14,18 +14,13 @@ DROPBEAR_KEYS = n
...
 PACKAGES += zlib
 PACKAGES += ncurses
 PACKAGES += readline
-PACKAGES += nano
-PACKAGES += screen
 
 PACKAGES += procServ
+PACKAGES += libmnl
 PACKAGES += ethtool
-PACKAGES += i2c-tools
```
[the remark this answers is not in the document]
Comment (EmilioPeJu) [reply rootfs/CONFIG:28]: Nope, the IPMI checks for the FMC eeprom are done by a python script, not this, I suspect it was added for testing purposes (e.g. scanning the I2C bus in search of devices)

File: kernel/zynq.config
[the remark this answers is not in the document]
Comment (EmilioPeJu) [reply kernel/zynq.config:1]: DRM is graphics related

File: rootfs/network/default.script [hunk 3]
```diff
@@ -71,15 +71,12 @@ case "$1" in
             fi
         fi
 
-        if [ -n "$timesrv" ]; then
-            echo "Recreating /etc/ntp.conf from $timesrv"
+        if [ -n "$ntpsrv" ]; then
+            echo "Recreating /etc/ntp.conf from $ntpsrv"
```
[the remark this answers is not in the document]
Comment (EmilioPeJu) [reply rootfs/network/default.script:75]: will do

File: rootfs/network/parse-config (hunk 1 above)
Comment (EmilioPeJu) [reply rootfs/network/parse-config:148]: it means we don't have some extra information files created by ntpd,I don't think anyone cared about the information anyways

**Pull request merged**